content/22.vaccines.md

```diff
@@ -171,7 +171,66 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+Despite regulatory, safety, and efficacy concerns, pre-orders for 1 billion doses of the Sputnik V were reported within days of the vaccine's approval in Russia [@doi:ft7j].
+Almost a month later, the phase I/II trial data was published [@doi:gg96hq].
+
+In the phase I/II trial study conducted between late June and early August 2020, 76 participants (18-60 years old) were split into two groups of 38 participants, which were non-randomised in two hospitals in Russia. 
+In phase I, 9 patients received rAd26 and 9 patients received rAd5-S to assess safety over 28 days. 
+In phase II, at least 5 days post the completion of phase I, 20 patients received a prime-boost vaccination of rAd26-S on day 0 and rAd5-S on day 2, which was administered intramuscularly.
+The phase I/II trial reported that both vaccines were deemed safe and well tolerated. 
+The most common adverse events reported were mild with pain at the injection site (58%), hypothermia (50%), headaches (42%), fatigue (28%), and joint and muscle pain (24%). 
+Seroconversion was observed in all participants three weeks post the second vaccination (day 42) and all participants produced antibodies to the SARS-CoV-2 glycoprotein. 
+Receptor binding domain-specific IgG were high (14,703 and 11,143 respectively), indicating a sufficient immune response from both the frozen and lyophilized versions of the vaccine. 
+Three weeks post the second vaccination, the virus-neutralizing geometric mean antibody titers were 49.25 and 45.95 from the frozen and lyophilized vaccines respectively.
+At 28 days, median cell proliferation of 1.3% CD4+ and 1.1% CD8+ were reported for the lyophilized vaccine and 2.5% CD4+ and 1.3% CD8+ for the vaccine stored frozen. 
+These results indicated that both forms of Sputnik V appeared to be safe and induce a humoral and cellular response to the vaccines [@doi:gg96hq], which may be robust enough to subsist and not wane rapidly [@doi:ghx7xz]. 
+
+A press release on November 11th reported positive results from an interim analysis of the phase III Sputnik V trials, which reported 92% efficacy in 16,000 volunteers [@doi:10.1126/science.abf6791].
+However, this was only two days after both Pfizer and BioNTech reported that their vaccines had an efficacy over 90%, which led to significant speculation about the Russian findings for a myriad of reasons including the lack of a published protocol and the “reckless” approval of the vaccine in Russia months prior to the publication of the interim results of the phase III trial [@doi:10.1126/science.abf6791; @doi:10.1038/d41586-020-03209-0].
```

Comment:
```suggestion
However, this release came only two days after both Pfizer and BioNTech reported that their vaccines had an efficacy over 90%, which led to significant skepticism of the Russian findings for a myriad of reasons including the lack of a published protocol and the “reckless” approval of the vaccine in Russia months prior to the publication of the interim results of the phase III trial [@doi:10.1126/science.abf6791; @doi:10.1038/d41586-020-03209-0].
```
Does skepticism work here? Some of the concerns listed don't seem speculative to me, if that makes sense!

Reply:
Yes, skepticism is better here